app/routes/session.py

```diff
@@ -129,9 +129,12 @@ def get_sign_out():
     """
     logout_user()
 
-    account_service_log_out_url = cookie_session.get("account_service_log_out_url")
-    if account_service_log_out_url:
-        return redirect(account_service_log_out_url)
+    redirect_url = request.args.get("redirect_url") or cookie_session.get(
```

Comment:
Not sure about redirect_url, I feel this loses context of what it is. sign_out_url?

Reply:
No longer applicable